src/js/tracks/text-track-display.js

```diff
@@ -279,6 +278,12 @@ class TextTrackDisplay extends Component {
 
     window.WebVTT.processCues(window, cues, this.el_);
 
+    if (this.player_.textTrackSettings === false) {
```

Comment:
I gave this a try and `this.player_.textTrackSettings` is undefined and not false when the TextTrackSettings component is not added. You might want to use something like [player.getChild('TextTrackSettings')](http://docs.videojs.com/component#getChild) to check for it.

Reply:
oh, thanks for pointing this out. We generally do loose checking so, just 
```js
if (!this.player_.textTrackSettings) {
  // ...
}
```
should be fine

Reply:
Yep, either works for me